packages/typescript-estree/src/parseSettings/createParseSettings.ts

```diff
@@ -0,0 +1,201 @@
+import debug from 'debug';
+import { sync as globSync } from 'globby';
+import isGlob from 'is-glob';
+
+import type { CanonicalPath } from '../create-program/shared';
+import {
+  ensureAbsolutePath,
+  getCanonicalFileName,
+} from '../create-program/shared';
+import { inferSingleRun } from '../inferSingleRun';
+import type { TSESTreeOptions } from '../parser-options';
+import type { MutableParseSettings } from './index';
+import { warnAboutTSVersion } from './warnAboutTSVersion';
+
+const log = debug(
+  'typescript-eslint:typescript-estree:parser:parseSettings/createParseSettings',
```

Comment:
```suggestion
  'typescript-eslint:typescript-estree:parser:parseSettings:createParseSettings',
```

I think that it has to be colons instead of slashes.
IDK if it would break anything to use slashes, but probably would cos `debug` is super simple